install.html

```diff
@@ -296,10 +296,14 @@ <h1 id="install"><i class="fa fa-wrench"></i>Installing the Hypervisor</h1>
                                 <li>Navigate to <a target="_blank" href="https://www.virtualbox.org/wiki/Downloads">generic download page</a> and download VirtualBox Extension Pack</li>
                                 <li>After downloading, double-click on the Extension Pack to install.</li>
                             </ol>
+                            <p>Issues in downloading or installing Oracle VirtualBox? Visit their <a href="https://www.virtualbox.org/manual/ch02.html#installation_windows">manual</a> for further instructions.</p>
                         </details>
                         <details class="install-hypervisor">
                             <summary>VMware Player & Pro</summary>
-                            VMware Player & Pro
+                            <ol>
+                                <li>VMware Player & Pro</li>
+                            </ol>
```

Comment:
Is this one-item oldered list not redundant title (what already exists as the <summary>? Why is this a list item btw?

Reply:
It was a placeholder for the installation steps, replaced with first actual step now